Automattic/jetpack
# Boost: Add edge-cache header on WP Cloud

## Proposed changes:
* If it's a WP Cloud site, add a header that helps cache the minified file.

### Other information:


## Jetpack product discussion
pfFwAW-nW-p2#comment-245

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Install it on a WPCOM site.
* Enable minify JS/CSS.
* Check the header on `_jb_static` files that loads.
* `A8c-Edge-Cache: cache` header should be present.
